src/ucp/core/ucp_ep.c

```diff
@@ -253,6 +253,8 @@ ucs_status_t ucp_ep_create(ucp_worker_h worker,
     uint64_t dest_uuid;
     ucp_ep_h ep;
 
+    UCS_STATIC_ASSERT(sizeof(*ep_p) <= 160); /* 10 * 16 bytes */
```

Comment:
1. pls put all these in standalone test, that will be compiled by jenkins only in release mode build
2. what is the meaning of the comment "10 * 16 bytes"?

Reply:
10*16 - it is hint for developer who will going to add new values to object: limitation is rounded by 16 bytes, and in case if object size should be reasonable increased then next limit should be rounded by 16 bytes too

Reply:
why 16 bytes?

Reply:
the minimal object size delta without significant performance impact, most of allocation calls returns pointers rounded by 16 bytes, and increasing size inside of 16-bytes limit should not affect performance